internal/bcdb/transaction_processor_test.go

```diff
@@ -565,6 +569,25 @@ func TestTransactionProcessor(t *testing.T) {
 		env := newTxProcessorTestEnv(t, cryptoDir, conf)
 		require.Nil(t, env)
 	})
+
+	t.Run("get cluster status", func(t *testing.T) {
+		cryptoDir, conf := testConfiguration(t)
+		require.NotEqual(t, "", cryptoDir)
+		defer os.RemoveAll(conf.LocalConfig.Server.Database.LedgerDirectory)
+		env := newTxProcessorTestEnv(t, cryptoDir, conf)
+		defer env.cleanup()
+
+		setupTxProcessor(t, env, worldstate.DefaultDBName)
+
+		require.Eventually(t, func() bool {
```

Comment:
Isn't it possible that leader will be elected fast enough and this condition never become true?

Reply:
This condition asserts that the current node is the leader; that is when IsLeader() returns nil, the current node is leader, so it is fine.

Reply:
I see